package/yast2-installation-control.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Mon Jun 26 11:09:53 CEST 2017 - shundhammer@suse.de
+
+- Allow different mount point for home partition (Fate#323532)
```

Comment:
NP: I'd add `(a workaround to allow defining data partitions)` so it's clear that there is no intention to have real home directories somewhere else than in `/home`. (The Fate details might not be accessible for public.)